.editorconfig

```
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Hey @rjsrams, thanks for the contribution.
Since this project is mostly Golang files, can you make this change? This way we get closer to lint results.

```suggestion
indent_style = tab
indent_size = 4
```

---

Done! Updated .editorconfig as suggested to use tabs for Go files.